app/chrome/panel.html

```diff
@@ -3,14 +3,16 @@
 
 <head>
   <title>COBI.bike DevKit Simulator</title>
-  <meta http-equiv="Content-Security-Policy" content="default-src *; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://maps.googleapis.com ">
+  <meta http-equiv="Content-Security-Policy" content="default-src *; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval' https://ssl.google-analytics.com https://maps.googleapis.com https://www.googletagmanager.com https://www.google-analytics.com">
   <link rel="stylesheet" href="style.css" media="all">
   <!--
       - JavaScript and HTML must be in separate files: see our Content Security
       - Policy documentation[1] for details and explanation.
       -
       - [1]: https://developer.chrome.com/extensions/contentSecurityPolicy
     -->
+    <!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
There is no "gtag.js" file in the project, is this just an indication for something else? or should it read "analytics.js"?